lib/splash.js

```diff
@@ -3,6 +3,8 @@ var _ = require("lodash");
 var level = 0;
 var _logger;
 
+var KEYS_TO_HIDE = ["pass", "secret"];
```

Comment:
This mean in .yml config of application that are using node-tech-logger, we are okay ?